### ISSUE TITLE
Feature Request: Not delete .git files/folders

### ISSUE THREAD

**Issue by jkl3848** [hyperbrew/bolt-uxp#29]
When running create-bolt-uxp, it clears out the entire folder. This wasn't a huge inconvenience for me, except that I have already init the folder to my git repo. Creating the project killed the link and I had to reconnect it. It's not a huge deal, only takes a couple minutes, but it is an inconvenience nonetheless.

**Comment by Klustre** [comment]
Duplicate of #12

**Comment by jkl3848** [comment]
> Duplicate of #12 

True. I dont mind the clearing of the folder. Its think its good to clean up before the project gets created. But I'd want to keep the git files. Maybe it's not worth being separate, it could just be my preference.

**Comment by justintaylor-dev** [comment]
Updated 1.0.0 will prompt you if a folder exists and you can choose to overwrite or pick another name.

92b1a53f0fb6089b68bb113f0554d360af83b342

You should init git after creation
